src/components/document-metadata-modal.js

```diff
@@ -167,20 +182,9 @@ function DocumentMetadataModal({
         allowedOpenContribution: canRestrictOpenContribution ? allowedOpenContribution : initialDocumentMetadata.allowedOpenContribution
       };
 
-      const actualTemplateDocumentId = determineActualTemplateDocumentId({
-        mode,
-        documentToClone,
-        useTemplateDocument,
-        defaultTemplateDocumentId
-      });
-
-      const targetRoomId = determineTargetRoomId({
-        mode,
-        initialDocumentMetadata,
-        documentToClone,
-        cloningStrategy,
-        cloningTargetRoomId
-      });
+      if (targetRoomId) {
```

Comment:
This will be changed/expanded with https://educandu.atlassian.net/browse/EDU-818 I presume.

Reply:
:+1: